aio_pika/pool.py

```diff
@@ -20,18 +20,19 @@ class Pool:
     __slots__ = (
         'loop', '__max_size', '__items',
         '__constructor', '__created', '__lock',
-        '__constructor_args'
+        '__constructor_args', '__close_lock'
     )
 
     def __init__(self, constructor: ConstructorType, *args,
                  max_size: int = None, loop: asyncio.AbstractEventLoop = None):
         self.loop = loop or asyncio.get_event_loop()
         self.__max_size = max_size
-        self.__items = asyncio.Queue(loop=self.loop)
+        self.__items = asyncio.Queue()
         self.__constructor = constructor
         self.__constructor_args = args or ()
         self.__created = 0
-        self.__lock = asyncio.Lock(loop=self.loop)
+        self.__lock = asyncio.Lock()
+        self.__close_lock = asyncio.Lock()
```

Comment:
I guess this is for #269. Maybe remove within current PR?

Reply:
Nope, commits from this PR will be removed from #269

Reply:
I mean this attribute is not used.

Reply:
Yes. Done

Reply:
`__close_lock` remained in `__slots__`.